Neuron/AppDelegate.swift

```diff
@@ -27,7 +27,7 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
         GuideService.shared.register()
         AuthenticationService.shared.register()
         SensorsAnalytics.configureSensors()
-        _ = TransactionStatusManager.manager
+        _ = TxStatusManager.manager
```

Comment:
This style for initializing the shared manager is very strange and confusing. Consider adding a `register()` func (move `init` logic into it) to this kind of manager classes and call them explicitly instead of referring to manager like this.